amethyst_utils/src/app_root_dir.rs

```diff
@@ -0,0 +1,18 @@
+use std::env;
+use std::fs;
+
+// Returns the cargo manifest directory for debug builds or the directory in which the current
+// executable resides for release builds, traversing a symlink if necessary.
+pub fn application_root_dir() -> String {
+    if env::var("PROFILE").unwrap() == "release" {
```

Comment:
You should definitely not use unwrap here. If you run the executable directly ./gamename it will panic
If "PROFILE" is set to either release or debug, use CARGO_MANIFEST_DIR.
Otherwise, that means the game was executed using ./gamename or by clicking on it (verify that's its the case).

Reply:
I was assuming that "PROFILE" was set at compilation for the binary.  I see now that it's an env var set by cargo that won't be set when run without it.  Silly mistake :/